src/components/Form/Form.js

```diff
@@ -40,6 +40,7 @@ export default function Form({ categories, onSubmit }) {
     const form = event.target
     if (isValidInput()) {
       onSubmit({ ...values, hours: Number(values.hours) })
+      localStorage.setItem('values', values)
```

Comment:
Which values you want to store? Values could be everything. Maybee you find more precise name for the variable.